addons/project/models/project.py

```diff
@@ -186,12 +186,18 @@ def default_get(self, flds):
         string='Members')
     is_favorite = fields.Boolean(compute='_compute_is_favorite', inverse='_inverse_is_favorite', string='Show Project on dashboard',
         help="Whether this project should be displayed on the dashboard or not")
-    label_tasks = fields.Char(string='Use Tasks as', default='Tasks', help="Gives label to tasks on project's kanban view.")
+    label_tasks = fields.Char(string='Use Tasks as', default='Tasks', help="Tasks of this project will be labelled this way in the Project Dashboard.")
     tasks = fields.One2many('project.task', 'project_id', string="Task Activities")
     resource_calendar_id = fields.Many2one(
         'resource.calendar', string='Working Time',
         default=lambda self: self.env.user.company_id.resource_calendar_id.id,
         help="Timetable working hours to adjust the gantt diagram report")
+    project_type = fields.Selection([
+            ('internal', _('Internal Project')),
+            ('portal', _('Customer Project'))
```

Comment:
Why `Customer Project` and not `Partner Project` ?

You may want manage project with your suppliers/vendors, don't you think ?